packages/@react-native-windows/tester/src/js/examples/Text/TextExample.windows.js

```diff
@@ -451,6 +451,37 @@ function AllowFontScalingExample(props: {}): React.Node {
   );
 }
 
+function MaxFontSizeMultiplierExample(props: {}): React.Node {
```

Comment:
Because these example files changed, the snapshots are failing for the e2etest-app tests. You'll need to update the snapshots for these to pass.